src/MyContext.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Always give your files meaningful names. In this case **UserContext.js** would have been a good option.

Reply:
please do refactor this